sdk/rust/src/entity_id.rs

```diff
@@ -93,7 +106,7 @@ impl EntityId {
         EvmAddress::try_from(self).map(|it| it.to_string())
     }
 
-    pub(crate) fn generate_checksum(entity_id_string: &String, ledger_id: LedgerId) -> Checksum {
+    pub(crate) fn generate_checksum(entity_id_string: &String, ledger_id: &LedgerId) -> Checksum {
```

Comment:
I just noticed this one, so, sorry for not mentioning it sooner, but `entity_id_string` should be an `&str` here